stdlib/public/core/KeyPathIterable.swift

```diff
@@ -104,6 +104,13 @@ extension Array : KeyPathIterable {
   }
 }
 
+extension Array.DifferentiableView : KeyPathIterable {
```

Comment:
I don't think you need a special implementation here. Derived conformances should just do it for you when you declare `: KeyPathIterable` at `DifferentiableView`'s declaration site.

Reply:
done